homeassistant/components/hatchrest/translations/en.json

```diff
@@ -0,0 +1,20 @@
+{
+    "config": {
```

Comment:
Please remove the translations folder. The translation string to directly in `strings.json`